ironfish/src/metrics/metricsMonitor.ts

```diff
@@ -19,6 +19,7 @@ export class MetricsMonitor {
 
   readonly mining_newBlockTemplate: Meter
   readonly chain_newBlock: Meter
+  readonly chainDatabaseSize: Gauge
```

Comment:
```suggestion
  readonly chain_databaseSize: Gauge
```

Reply:
I will change it, curious about the reason for this particular name convention.